### PR TITLE
Disable robots in the ITB

### PR DESCRIPTION
**Pull request by brianhlin**
I've created [this branch](https://github.com/opensciencegrid/docs/tree/itb.no_robots), which [deployed](https://travis-ci.org/opensciencegrid/docs/builds/301655111) to [docs-itb](https://opensciencegrid.github.io/docs-itb/) and you can see the relevant [meta tag](https://developers.google.com/search/reference/robots_meta_tag#using-the-robots-meta-tag) in the source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ Getting started
 
 1. From the root directory of your parent repository, copy the template to your travis configuration:
 
-        $ cp ci/travis.template.yml .travis.yml
-        $ cp ci/travis.template.env .travis.env
+        $ cp ci/travis.yml.template .travis.yml
+        $ cp ci/travis.env.template .travis.env
```
Comment (matyasselmeci) [inline README.md:20]: Any reason for the rename? The nice thing about keeping the original extension is that syntax highlighting continues to work.

No big deal, but if you go through with it, check the links and references elsewhere on this page. Line 22 still points to the old name.

Comment (matyasselmeci) [reply README.md:20]: My bad, didn't realize it was already named that way. Still, check line 22.

**Comment by brianhlin** [comment]
Should be fixed, ready for re-review @matyasselmeci

**Comment by matyasselmeci** [comment]
LGTM.

**Pull request merged**